app/steps/ui/deceased/anydeceasedchildren/index.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const ValidationStep = require('app/core/steps/ValidationStep');
+const FormatName = require('app/utils/FormatName');
+const content = require('app/resources/en/translation/deceased/anydeceasedchildren');
+const {get} = require('lodash');
+
+class AnyDeceasedChildren extends ValidationStep {
+
+    static getUrl() {
+        return '/any-deceased-children';
+    }
+
+    getContextData(req) {
+        const ctx = super.getContextData(req);
+        const formdata = req.session.form;
+        ctx.deceasedName = FormatName.format(formdata.deceased);
```

Comment:
again action method and delete ctx.entries

Reply:
Technically yes, but I'll remove this code in this PR otherwise I'll have to add the tests as well.
This is the next question and this code will go back in with the next PR.